### PR TITLE
docs - make installation easier to understand for macOS and Ubuntu

### PR DESCRIPTION
**Pull request by SmartKeyerror**
## Here are some reminders before you submit the pull request
- [ ] Add tests for the change
- [ ] Document changes
- [ ] Communicate in the mailing list if needed
- [ ] Pass `make installcheck`
- [ ] Review a PR in return to support the community

### CODE REVIEW

File: README.Linux.md [hunk 1]
```diff
@@ -43,15 +43,15 @@ Use dependency script for CentOS.
   `export DEBIAN_FRONTEND=noninteractive`
 
   ```bash
-    ./README.Ubuntu.bash
+    sudo ./README.Ubuntu.bash
```
Comment (higuoxing) [inline README.Linux.md:46]:
```suggestion
  sudo ./README.Ubuntu.bash
```

**Review by higuoxing — COMMENTED** [review]
Thanks for contributing! Here are a couple of comments.

File: README.Linux.md [hunk 2]
```diff
@@ -77,7 +77,8 @@ then run command `ldconfig`.
    chmod 600 ~/.ssh/authorized_keys
    ```
 
-1. Verify that you can ssh to your machine name without a password
+1. Verify that you can ssh to your machine name without a password.
+   You can use `hostname` to get the hostname of your machine.
 
    ```bash
     ssh <hostname of your machine>  # e.g., ssh briarwood
```
Comment (higuoxing) [inline README.Linux.md:84]: I would suggest putting the hint into the comment.
```suggestion
    ssh <hostname of your machine>  # e.g., ssh briarwood (You can use `hostname` to get the hostname of your machine)
```

File: README.macOS.md [hunk 3]
```diff
@@ -61,7 +68,8 @@ EOF
 
 Currently, the GPDB utilities require that it be possible to `ssh` to localhost
 without using a password.  To verify this, the following command should succeed
-without erroring, or requiring you to enter a password.
+without erroring, or requiring you to enter a password. You can use `hostname` to
+get the hostname of your machine.
 
 ```
 ssh <hostname of your machine>  # e.g., ssh briarwood
```
Comment (higuoxing) [inline README.macOS.md:75]: Ditto.

**Comment by SmartKeyerror** [comment]
@higuoxing Thanks for your advising, and I have re-edited, please review it again.

**Comment by higuoxing** [comment]
Merged, thanks!

**Pull request merged**